data/human/free worlds start.txt

```diff
@@ -781,7 +781,7 @@ mission "FW Plasma Testing 1B"
 		log "Assisted a Kraz Cybernetics engineer known as Barmy Edward with a test of a new and extremely powerful plasma cannon turret."
 		conversation
 			scene "scene/loc7"
-			`When you enter the hangar, the Window-Blind has just returned, and a team of engineers are looking it over. Barmy Edward puts down his work to come over and debrief you. "How did the turrets perform?" he asks.`
+			`When you enter the hangar, the Window-Blind has just returned, and a team of engineers is looking it over. Barmy Edward puts down his work to come over and debrief you. "How did the turrets perform?" he asks.`
```

Comment:
Are you sure about this one? Plural noun `engineers` needs a plural verb

Reply:
“A team of engineers” is singular; the “is” applies to “a team” and “of engineers” is a description of said team.

Reply:
It might be a little pedantic, but the subject in this sentence is team, not engineers. So the original sentence without modifiers reads as "a team are looking it over".

Reply:
My take would be team of engineers is a collective noun. Since they're looking it over as a collective group, the verb would be singular. If the team are "running in different directions," something they don't do as a group, then the verb would be plural.